packages/identity-sdk/src/viem-sdk.ts

```diff
@@ -0,0 +1,215 @@
+import {
+  Address,
+  PublicClient,
+  WalletClient,
+  parseAbi,
+  SimulateContractParameters,
+  WalletActions,
+} from "viem";
+import { waitForTransactionReceipt } from "viem/actions";
+import { z } from "zod";
+import { compressToEncodedURIComponent } from "lz-string";
+
+import { Envs, FV_IDENTIFIER_MSG2 } from "./constants";
+
+const identityV2ABI = parseAbi([
+  "function addWhitelisted(address account)",
+  "function removeWhitelisted(address account)",
+  "function isWhitelisted(address account) view returns (bool)",
```

Comment:
getWhitelistedRoot should be used in general.
also devs should use it. if root is not address zero then account is whitelisted or connected to a whitelisted account.
Its the dev decision how to handle connected accounts.

Reply:
done now